### DISCUSSION TITLE
[Decode Dot Transaction] why can i decode polkadot transaction on polkadot.js ?

### DISCUSSION THREAD

**Discussion by weixuefeng** [trustwallet/wallet-core#3540] [Q&A]
i can not decode [transaction](https://github.com/trustwallet/wallet-core/blob/1382e3c8ac6d8e956e25c0475039f6c3988f9355/tests/chains/Polkadot/SignerTests.cpp#L67) on [polkadot.js](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2F1rpc.io%2Fdot#/extrinsics/decode).
but i can decode transaction by subwalelt.

**Comment by satoshiotomakan** [comment]
Hi @weixuefeng, I'll take a look the reason.

**Comment by weixuefeng** [comment]
when i use js sdk sign i got
`0x2d028400ea2ca7e445d363a8114ba3e471c8a87aef04af19abfd096c1a3287c4d3eb87950060429cc22c7b73d27cae1d9b4ede663aad7a8d8e7d3a6b9b1eb778fb6595a3acb702dcc2e986a8f52b5f29a6e18b72efb728780a03cd4673339b5d763d97ac0556033800050000d533f5f76d2a114a42575e9293b8e0468415d131a1fa734ec637214a1bd521db04`

i can split transaction as:
```
// 0x 2d0284 --> something
// 00 --> some prefix
// ea2ca7e445d363a8114ba3e471c8a87aef04af19abfd096c1a3287c4d3eb8795  // signer
// 00 // signature type
// 60429cc22c7b73d27cae1d9b4ede663aad7a8d8e7d3a6b9b1eb778fb6595a3acb702dcc2e986a8f52b5f29a6e18b72efb728780a03cd4673339b5d763 // signature
// d97ac0556033800  -->era, what's the rule ?
// 0500 -->callIndex
// 00 -->tip
// d533f5f76d2a114a42575e9293b8e0468415d131a1fa734ec637214a1bd521db -->dest
// 04  -->value
```

and when i use wallet core sign a transaction, i got
`0x310284e966e51a2a73179637494b0647581963185f59e9b00eaaecb0d4fa0163a89b5b00a2bf1619d72a9e6795dfc899ffa7c5a81759b4069548871485df867591411d3cd49998a042259b13d57a4078cb017b3fb08871e25cce458cedea97c7414a620f520000a10f000500d533f5f76d2a114a42575e9293b8e0468415d131a1fa734ec637214a1bd521dba10f`

i split it
```
//0x 310284 --> something
// no prefix? ----> is it a problem?
//e966e51a2a73179637494b0647581963185f59e9b00eaaecb0d4fa0163a89b5b signer
//00 signature type
//a2bf1619d72a9e6795dfc899ffa7c5a81759b4069548871485df867591411d3cd49998a042259b13d57a4078cb017b3fb08871e25cce458cedea97c741
//4a620f520000a10f00
// 0500 callIndex?
//d533f5f76d2a114a42575e9293b8e0468415d131a1fa734ec637214a1bd521db
//a10f

```
